.env.example

```diff
@@ -14,4 +14,7 @@ CONTENTFUL_ACCESS_TOKEN=
 # Your current space Content Preview API access token: https://www.contentful.com/developers/docs/references/content-preview-api/
 CONTENTFUL_PREVIEW_ACCESS_TOKEN=
 
-# Create the following .env files: env.local, env.staging, env.production
+#HOME PAGE SLUG
+CONTENTFUL_HOMEPAGE_SLUG=home
```

Comment:
I think it's not the purpose of a .env to hold such variables, since this is not a variable to keep secret we can put this  in a regular config
@ceneva what are you're thoughts?

Reply:
It's just so that we have one global place to keep all the variables. It's not perse to keep it secret.

Reply:
However it might be a good idea to keep a separate config for non-sensitive data and combine both, since you can store this info in git then, you can create a folder to store different env configurations in.